docs/docs/router.md

```diff
@@ -218,7 +218,7 @@ import { NavLink, routes } from '@redwoodjs/router'
 const MainMenu = () =>
   <ul>
     <li>
-      <!-- When match "/" -->
+      <!-- Will match "/" -->
```

Comment:
The original wasn't clear, but I think the spirit of it is lost if we replace it with just "will". I think "when" was supposed to be short for something like `adds "activeLink" when the URL matches "/"`. Maybe we could do `will add "activeLink" when the URL matches "/"`?

Reply:
https://github.com/redwoodjs/redwood/pull/10401